src/Network/Haskoin/Store/Data.hs

```diff
@@ -276,6 +304,14 @@ instance BinSerial Balance where
         putWord64be c
         putWord64be t
 
+    binDeserial net = do a <- binDeserial net
+                         v <- getWord64be
+                         z <- getWord64be
+                         u <- getWord64be
+                         c <- getWord64be
+                         Balance a v z u c <$> getWord64be
+
+
```

Comment:
I would say in this case the same applies, as the order of the parameters in the constructor is the same as in the serialized version:

```haskell
Balance
    <$> binDeserial net
    <*> getWord64be
    <*> getWord64be
    <*> getWord64be
    <*> getWord64be
    <*> getWord64be